### ISSUE TITLE
[Bug]: "Macy's" Sponsored shortcut redirects to "Access denied"

### ISSUE THREAD

**Issue by SoftVision-LorandJanos** [mozilla-mobile/fenix#24094]
### Steps to reproduce

1. Have the Sponsored shortcuts feature enabled.
2. Tap on the "Macy's" Sponsored shortcut and observe the page.

### Expected behaviour

Macy's page opens correctly.

### Actual behaviour

Tapping on "Macy's" redirects to "Access denied".

### Device name

Google Pixel 4

### Android version

Android 12

### Firefox release type

Firefox Nightly

### Firefox version

99.0a1

### Additional information

![Screenshot_20220304-093438](https://user-images.githubusercontent.com/48829350/156723185-11cc5e3c-2fd7-4571-a739-a4628eaa2e9d.png)

1.0.2210 (Build 1)
AC: 99.0.20220228143604,
5330648f35
GV: 99.0a1-20220228090129
AS: 91.1.0
debud build

**Comment by lwa-moz** [comment]
@SoftVision-LorandJanos would you be able to provide the URL that the tile tried to resolve?

**Comment by SoftVision-LorandJanos** [comment]
@lwa-moz  sure: https://www.macys.com/?cm_mmc=Google_AdMarketPlace-_-Privacy_Tile-_-1646407623400800000_Broad-_-kclickid__kenshoo_clickid_&m_sc=sem&m_sb=Admarketplace&m_tp=Search&m_ac=Admarketplace&m_ag=Tile&m_cn=Privacy&m_pi=kclickid__kenshoo_clickid__1646407623400800000

**Comment by SoftVision-CosminMuntean** [comment]
I just managed to reproduce this on Nike Sponsored tile on FR region using a VPN service. After I clicked the Nike Sponsored tile I am redirected to an "Access denied" page.  Here is the link:  https://www.nike.com/fr/?cp=19040524063_search_1646831752400400006

However, I have noticed that after disconnecting from the VPN service, the link can be successfully opened in the browser. Probably this issue might be caused by the VPN service?
